tests/python/YapfBearTest.py

```diff
@@ -24,6 +24,13 @@ def test_valid(self):
                             ["x = {  'a':37,'b':42,\n", "'c':927}\n", '\n',
                              "y = 'hello ''world'\n"], valid=False)
 
+    def test_eof_handling(self):
+        self.check_validity(self.uut, [], valid=True)
+        self.check_validity(self.uut, [''], valid=True)
+        self.check_validity(self.uut, ['a = 2\n'], valid=True)
+        self.check_validity(self.uut, ['a = 2'], valid=True)
```

Comment:
yeah I thought it would be intended that it would do that but this is fine as well